kolibri/core/assets/src/core_app_constructor.js

```diff
@@ -77,10 +72,20 @@ module.exports = function CoreApp() {
       (require) => {
         require('intl');
         require('intl/locale-data/jsonp/en.js');
+        /**
```

Comment:
Moved this inside the require blocks, because `Intl` has to be defined when `vue-intl` is required.